lib/statsd/instrument.rb

```diff
@@ -38,10 +38,18 @@ def generate_metric_name(name, callee, *args)
       # Generates the tags for an instrumented method.
       # @private
       # @return [Array[String]]
-      def generate_tags(tags, callee, *args)
+      def generate_tags(tags, callee, result=nil, *args)
```

Comment:
I need to fix this, probably requires a bool `use_result = false` for real nilable result objects.